src/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -1781,9 +1781,7 @@ public abstract partial class DbConnection : System.ComponentModel.Component, Sy
     {
         protected DbConnection() { }
         [System.ComponentModel.DefaultValueAttribute("")]
```

Comment:
@corivera can you please verify that these APIs should be included for both netcoreapp and uap?